src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/targets/Microsoft.Android.Sdk.Application.targets

```diff
@@ -15,8 +15,6 @@ This file contains targets specific for Android application projects.
     <UseAppHost>false</UseAppHost>
     <!-- see: https://github.com/xamarin/xamarin-macios/blob/a6eb528197854c074d9dd5847328c096890337be/dotnet/targets/Xamarin.Shared.Sdk.props#L38-L52 -->
     <_RuntimeIdentifierUsesAppHost>false</_RuntimeIdentifierUsesAppHost>
-    <RunCommand>dotnet</RunCommand>
-    <RunArguments>build &quot;$(MSBuildProjectFullPath)&quot; -target:Run --configuration &quot;$(Configuration)&quot;</RunArguments>
```

Comment:
Note that before, we had no way to parse all the `-p` arguments passed to `dotnet run` through to the underlying `Run` target. We had `$(Configuration)` hardcoded here, but that was the only one that worked.